jaxb/src/main/java/feign/jaxb/JAXBContextFactory.java

```diff
@@ -73,6 +74,12 @@ private JAXBContext getContext(Class<?> clazz) throws JAXBException {
     return jaxbContext;
   }
 
+  private void preloadContextCache(List<Class<?>> classes) throws JAXBException {
+    for (Class<?> clazz : classes) {
```

Comment:
It will take a list of classes in the builder when the factory is initialized.
But i asume if a use case justify making it public that should not be a problem

Reply:
The list can be applied during building:
```java
JAXBContextFactory.build(List<Class<?>> classes)
```
Additional classes can then be loaded on demand.